eslint.config.mjs

```diff
@@ -105,6 +107,7 @@ export default [
       'no-shadow-restricted-names': 'error',
       '@typescript-eslint/no-unused-vars': 'warn',
       'react-hooks/exhaustive-deps': 'error',
+      '@typescript-eslint/no-unused-vars': noUnuseVars,
```

Comment:
Do we need to specify it explicitly? I thought `no-unused-vars` ignores underscore by default.

Reply:
No it doesn't